proto/p2p/p2p.proto

```diff
@@ -29,6 +29,7 @@ message Message {
     Ping ping = 11;
     Pong pong = 12;
     Handshake handshake = 13;
+    GetPeerList get_peer_list = 35;
```

Comment:
not sure what the best practice is for proto but it feels more natural to put it in numeric order

Reply:
Currently this proto definition groups based on the logical usage of the messages... I don't think there is a standard format for these... I know some people reserve the next unused message to keep the counter... But I don't think `reserved` works in a `oneof`

Reply:
I reserved the next field number.